cli/pcluster/config/mappings.py

```diff
@@ -477,6 +478,11 @@
         "s3_read_write_resource": {
             "cfn_param_mapping": "S3ReadWriteResource",  # TODO add validator
         },
+        "disable_hyperthreading": {
```

Comment:
Could you please add unit tests for this new parameter in the `test_section_cluster.py --> test_cluster_param_from_file`  ?

Reply:
Probably you could also add muliple tests in the `test_section_cluster.py --> test_cluster_section_to_file` since the disable_hyperthreading is strictly related to the extra_json and we can test the different combinations.

Reply:
I added tests to `test_cluster_param_from_file()` and `test_cluster_section_to_file()`